docs/docs/concepts/structured_outputs.mdx

```diff
@@ -152,3 +152,20 @@ ResponseFormatter(answer="The powerhouse of the cell is the mitochondrion. Mitoc
 For more details on usage, see our [how-to guide](/docs/how_to/structured_output/#the-with_structured_output-method).
 
 :::
+
+## Handling parsing errors
+
+In production, LLMs may return malformed outputs. While `with_structured_output()` doesn't provide automatic retry on parsing errors, you can use `OutputFixingParser` for this:
+
+```python
+from langchain.output_parsers import OutputFixingParser
```

Comment:
FYI we're not including this in langchain_v1 yet. We'll likely need to figure out how to have the same functionality exposed in v1.

Reply:
@eyurtsev what is the recommended approach in V1? `with_structured_output`?